homework/shelkovich_ilya/primitiveAndWrapperSpeed/src/main/java/com/tutrit/java/quickstart/service/Sum.java

```diff
@@ -0,0 +1,28 @@
+package com.tutrit.java.quickstart.service;
+
+public class Sum {
+
+    public int sumByPrimitives(int a, int b){
+        return a + b;
+    }
+
+    public Integer sumByWrapper(Integer a, Integer b){
+        return a + b;
+    }
+
+    public long millsToCalculatePrimitives(){
+        int i = 0;
+        while (i < 10_000_000){
+            sumByPrimitives(3,5);
+            i++;
+        } return System.currentTimeMillis();
+    }
+
+    public long millsToCalculateWrappers(){
```

Comment:
As you return mills, consider to get System.currentTimeMillis(); at the beginning of a method, then return difference between it and   System.currentTimeMillis(); in the end of the method.

Reply:
figured out how to make it work, thx